torch/_dynamo/utils.py

```diff
@@ -807,6 +821,18 @@ def get_compilation_metrics() -> List[Union[CompilationMetrics, BwdCompilationMe
     return list(_compilation_metrics)
 
 
+chromium_event_stack = ["__start__"]
+# Generate a unique id for this process, which we can use in scuba to filter down
+# to a single python run.
```

Comment:
It's weird that there's not already a canonical field or something like for this kind of use case